src/main/java/com/FlightPub/Controllers/IndexController.java

```diff
@@ -219,19 +219,20 @@ public String runSearch(@ModelAttribute BasicSearch search, Model model, HttpSes
     }
 
     @PostMapping("/advancedSearch")
-    public String runAdvancedSearch(@ModelAttribute BasicSearch search, Model model, HttpSession session)
-    {
+    public String runAdvancedSearch(@ModelAttribute BasicSearch search, Model model, HttpSession session) {
         model = addDateAndTimeToModel(model);
         List<Flight> flights;
-        List<SingleStopOver> flights1Stop = null;
-        List<MultiStopOver> flights2Stop = null;
+        List<StopOver> flights1Stop = null;
```

Comment:
Would is it easier to convert this to an array and just reference the indexes?

Reply:
Good Call

Reply:
Should be fixed now @WillPaterson

Reply:
Yeah, looks good 👍